src/servicebus/azext_servicebus/_help.py

```diff
@@ -0,0 +1,497 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps
+
+helps['servicebus'] = """
+    type: group
+    short-summary: Manage Azure ServiceBus namespace, queue, topic, subscription, rule and geo disaster recovery configuration - alias
```

Comment:
I believe the proper service name is 'Service Bus' not 'ServiceBus'.
Can you change this in the help file?

Reply:
Resolved